components/kyma-environment-broker/internal/appinfo/dto.go

```diff
@@ -0,0 +1,30 @@
+package appinfo
+
+import "time"
+
+type (
+	RuntimeDTO struct {
+		RuntimeID         string    `json:"runtimeId"`
+		GlobalAccountID   string    `json:"globalAccountId"`
+		SubAccountID      string    `json:"subaccountId"`
+		ServiceInstanceID string    `json:"serviceInstanceId"`
+		ServiceClassID    string    `json:"serviceClassId"`
+		ServiceClassName  string    `json:"serviceClassName"`
+		ServicePlanID     string    `json:"servicePlanId"`
+		ServicePlanName   string    `json:"servicePlanName"`
+		Status            DTOStatus `json:"status"`
+	}
+
+	DTOStatus struct {
```

Comment:
please, use one convention, DTO as suffix or prefix, see: RuntimeDTO and DTOStatus

Reply:
agree, fixed 👍